frontend/pages/topics/[slug].tsx

```diff
@@ -22,11 +36,24 @@ const TopicDetailPage: FC<TopicProps> = ({ topic }: TopicProps) => {
         {topic.description && <p dangerouslySetInnerHTML={{ __html: topic.description }} />}
         {topic.propositions &&
           topic.propositions.map((proposition) => (
-            <Link href={`/propositions/${proposition.slug}`} key={proposition.slug}>
+            <Link href={`/propositions/${proposition.slug}`} key={proposition.slug} passHref>
               <a>
-                <div>
-                  <p dangerouslySetInnerHTML={{ __html: proposition.summary }} />
-                </div>
+                <StyledTopicResultCard>
```

Comment:
Should we add the [`raised`](https://mui.com/api/card/#props) prop here? I think it looks better.